src/modules/emote_menu/styles/sidebar.module.css

```diff
@@ -0,0 +1,11 @@
+.sidebar {
+  width: 52px;
+  height: 100%;
+  overflow-y: scroll;
+  background-color: #191c23;
+  scrollbar-width: none;
+}
+
+.sidebar::-webkit-scrollbar {
```

Comment:
can you explain this? overflow-y is why you have a scrollbar, and it sounds like you need to turn that off.

Reply:
how are you meant to scroll otherwise?